Simulator/Test/test_matrix_math.py

```diff
@@ -9,17 +10,199 @@
 
 import sun_sensor_math as LA
 
+def pytest_addoption(parser):
```

Comment:
What does this do? I see it at the top of these test files.

Reply:
This is something I saw on a pytest example, but it can be taken out if it's not needed.  Some of this stuff I'm not sure I added though.

Reply:
Take it out and see if this breaks. I don't think we need it (not in any of the other files) Likewise with `@pytest.fixture` and `cmdopt`; we shouldn't need any of that weird stuff.

Reply:
Looks like it isn't necessary--nothing breaks